### ISSUE TITLE
Sync notes from other sources

### ISSUE THREAD

**Issue by passionate2023** [blackhole89/notekit#151]
Sync options needed from other sources

1) Sync Notes from https://syncthing.net/ 
2) Google Drive
3) Dropbox

Thanks

**Comment by sp1ritCS** [comment]
Just change your notesbase to a directory that is synced by whatever service you're using.

**Comment by passionate2023** [comment]
you mean the http path of the directory where the .txt files resides in the server?

**Comment by sp1ritCS** [comment]
No, all these cloud storage services offer some sort of sync client that synchronizes a directory with the cloud.

See
- https://nextcloud.com/install/
- https://www.dropbox.com/install
- https://support.google.com/drive/answer/10838124?hl=en

I'd advice against mounting a "cloud disk" as WebDAV / ftpfs / sshfs / etc. as if you don't have blazingly fast internet notekit probably blocks (and freezes) until the file has been saved)
